BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
What is the meaning of the `BUILD.bazel` file at the root? Does it represent the entire project?

Reply:
It represents all targets at the root of the project. In this case, specifically we use it to create a target for the LICENSE file that we can consume in other rules.